reddit_liveupdate/controllers.py

```diff
@@ -1090,3 +1092,38 @@ def GET_mediaembed(self, liveupdate, embed_index):
         }
 
         return pages.LiveUpdateMediaEmbedBody(**args).render()
+
+
+@add_controller
+class LiveUpdateAdminController(RedditController):
+    @validate(VAdmin())
+    def GET_happening_now(self):
+        candidate_thread_ids = queries.get_active_events()
+        candidate_threads = LiveUpdateEvent._byID(list(candidate_thread_ids),
+                return_dict=False)
+
+        candidate_threads = filter(lambda t: not (t.banned or
```

Comment:
nit: since this is in public code now, it might be a good time to clean up the indents to match our style guide.